x-pack/plugins/snapshot_restore/public/app/services/http/use_request.ts

```diff
@@ -30,7 +30,7 @@ export const sendRequest = async ({
   try {
     const response = await httpService.httpClient[method](path, body);
 
-    if (!response.data) {
+    if (typeof response.data === 'undefined') {
```

Comment:
Had to change this as a 200 response with empty string `''` was taken as an error.